<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanBuilderDialog</class>
 <widget class="QDialog" name="ScanBuilderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scan Parameters</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>bscan</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leLimitsBscanMin">
          <property name="text">
           <string>-2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="leLimitsBscanMax">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>volume</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leLimitsVolumeMin">
          <property name="text">
           <string>-2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="leLimitsVolumeMax">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLimitsOffsetX">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leLimitsOffsetY">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pbUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbExit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
